build/pipeline-releasebuild.yaml

```diff
@@ -46,14 +47,29 @@ steps:
     BuildPlatform: '$(BuildPlatform)'
     BuildConfiguration: '$(BuildConfiguration)'
     MsalClientSemVer: $(MsalClientSemVer)
-    Solution: 'Libs.sln'
+    Solution: 'LibsAndSamples.sln'
 
 # Run Post-build code analysis (e.g. Roslyn)
 - template: template-postbuild-code-analysis.yaml
 
 # Pack and sign all of the nuget packages
 - template: template-pack-and-sign-all-nugets.yaml
 
+- task: UseDotNet@2
```

Comment:
Shouldn't this run first?

Reply:
There was a note that I read that stated the signing tasks cannot run on .core 3 only 2, so I reset the version to 3 here after the signing. I am going to refactor all of these into a template so this will be removed

Reply:
That makes sense, I think @jennyf19 also encountered this problem.

Reply:
yes. it has to be version 2.x, you guys have it right, we had 2.0, and the error said to use 2.1, which doesn't work, but 2.x does.